akka-http-core/src/main/scala/akka/http/model/HttpEntity.scala

```diff
@@ -160,11 +160,11 @@ object HttpEntity {
     override def toStrict(timeout: FiniteDuration)(implicit fm: FlowMaterializer) =
```

Comment:
Yeah, but, AFAICS, the only way to do it is to add a `data.mapMaterialized(_ ⇒ ()))`, which incurs overhead *always*, even if I already pass in a `Source[ByteString, Unit]`.